fuzzing/fuzz_parser.cpp

```diff
@@ -39,14 +40,27 @@ fuzz_parser(string_view sv)
 extern "C"
 int
 LLVMFuzzerTestOneInput(
-    const uint8_t* data, size_t size)
+        const uint8_t* data, size_t size)
 {
+    if(size<1)
+        return 0;
+
+    parse_options popt;
+    popt.allow_comments=!!(data[0]&0x1);
+    popt.allow_trailing_commas=!!(data[0]&0x2);
+    popt.allow_invalid_utf8=!!(data[0]&0x4);
+    popt.max_depth= (data[0]>>3);
```

Comment:
hey that's really cool!!!

Reply:
Yep, only cool kids do bit shifts and stuff :-)

Reply:
LOL no... I mean, the mechanism for prodding the fuzzing tool to fill out the parse options

Reply:
Why are you not impressed with my coding skillz? I can do bitwise or, too!

On a more serious note, yes it is awesome to fill out input data from the fuzzer stream. Have a look at the curl fuzzer, which sets flags and then cuts different parts of the input data as the (fake) server response.